web/styles/zulip.css

```diff
@@ -2,10 +2,13 @@ body,
 html {
     width: 100%;
     height: 100%;
+    /* NOTE: Setting `overflow-x/y: hidden` on any element below this will be make
+       that element scrolling container of sticky element inside that element
+       and then the sticky behaviour of the element might not work. */
```

Comment:
I don't quite understand -- isn't every element below HTML?

Reply:
For example if you set `overflow-x` hidden on `.app`, it breaks the `sticky` behaviour of message header. Let me make this comment more clear.